docs/guides/tracks.md

```diff
@@ -1,44 +1,50 @@
-Tracks
-======
+# Tracks
 
 Text Tracks are a function of HTML5 video for providing time triggered text to the viewer. Video.js makes tracks work across all browsers. There are currently five types of tracks:
 
 - **Subtitles**: Translations of the dialogue in the video for when audio is available but not understood. Subtitles are shown over the video.
 - **Captions**: Transcription of the dialogue, sound effects, musical cues, and other audio information for when the viewer is deaf/hard of hearing, or the video is muted. Captions are also shown over the video.
 - **Chapters**: Chapter titles that are used to create navigation within the video. Typically they're in the form of a list of chapters that the viewer can click on to go to a specific chapter.
-- **Descriptions** (not supported yet): Text descriptions of what's happening in the video for when the video portion isn't available, because the viewer is blind, not using a screen, or driving and about to crash because they're trying to enjoy a video while driving. Descriptions are read by a screen reader or turned into a separate audio track.
-- **Metadata** (not supported yet): Tracks that have data meant for javascript to parse and do something with. These aren't shown to the user.
+- **Descriptions**: Text descriptions of what's happening in the video for when the video portion isn't available, because the viewer is blind, not using a screen, or driving and about to crash because they're trying to enjoy a video while driving. Descriptions are read by a screen reader or turned into a separate audio track. Captions and Subtitles take precedence of Descriptions when they are selected.
```

Comment:
There are three separate issues here:

1. Using the `default` attribute on a track, and how video.js handles the (error) case where there are multiple default tracks
2. What video.js does for [displaying tracks if more than one track is set to `showing`](https://github.com/videojs/video.js/blob/master/src/js/tracks/text-track-display.js#L117-140) (by any mechanism)
3. How the UI behaves (if you select one `kind` of track, it [disables any other tracks of the same kind](https://github.com/videojs/video.js/blob/master/src/js/control-bar/text-track-controls/text-track-menu-item.js#L81-93), but the display part doesn't support display of multiple `kind`s at once, as noted in 2). 

These three are obviously related, but in my mind are distinct. And now I've basically written the note myself! :-)